plugins/obs-filters/data/locale/en-US.ini

```diff
@@ -75,7 +75,9 @@ NoiseSuppress.Intensity="Suppression Intensity"
 NoiseSuppress.Method="Method"
 NoiseSuppress.Method.Speex="Speex (low CPU usage, low quality)"
 NoiseSuppress.Method.RNNoise="RNNoise (good quality, more CPU usage)"
-NoiseSuppress.Method.nvafx="NVIDIA Noise Removal (good quality, no CPU usage)"
+NoiseSuppress.Method.Nvafx="NVIDIA Noise Removal FX"
+NoiseSuppress.Method.Nvafx.Dereverb="NVIDIA Dereverb FX"
+NoiseSuppress.Method.Nvafx.Dereverb.Denoiser="NVIDIA Noise Removal + Dereverb FX"
```

Comment:
If using "Noise Removal" as the user-facing string, I'd use "Room Echo Removal" instead of "Dereverb". That aligns with the phrasing in both of these documents:
* https://docs.nvidia.com/deeplearning/maxine/audio-effects-sdk/index.html#afx-intro
* https://nvidia.frontify.com/d/uAobRitG8H8B/with-nvidia#/nvidia-broadcast-partner-branding-guidelines/messaging/feature-naming

The full string would then be:
> NVIDIA Noise Removal and Room Echo Removal

I don't think we need "FX" appended to each string.

Reply:
The SDK also uses Dereverb but ok, i've changed to Room Echo Removal.